sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2309,19 +2304,15 @@ object functions {
    * @group math_funcs
    * @since 3.3.0
    */
-  def ceil(e: Column, scale: Column): Column = withExpr {
-    UnresolvedFunction(Seq("ceil"), Seq(e.expr, scale.expr), isDistinct = false)
```

Comment:
I just realize that it maybe problematic in such cases, if some users happen to register a udf with the same name `ceil`

Reply:
Good point. If we want avoid this issue, it seems we should make the `built-in-only`, `udf-only`, `global` as you said.

Reply:
ok, if the goal of this PR is to replace `call_udf` with `call_function`, we can resolve this naming conflict issue in another PRs.